generated/niscope/niscope/session.py

```diff
@@ -4729,6 +4819,68 @@ def export_attribute_configuration_file(self, file_path):
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return
 
+    @ivi_synchronized
+    def get_ext_cal_last_date_and_time(self):
+        '''get_ext_cal_last_date_and_time
+
+        Returns the date and time of the last external calibration performed.
+
+        Note: The NI 4050 and NI 4060 are not supported.
+
+        Returns:
+            month (int): Indicates the **month** of the last calibration.
```

Comment:
based on the code only the docs are wrong

Reply:
I believe I was referencing nidcpower and nidmm metadata for this, and I assumed that both of those modules were correct in calling the output the month, even though it didn't look like that was what was happening on the code side of things. I'll fix that!

Reply:
Please create an issue for the incorrect names in nidcpower and nidmm

Reply:
Done in #1484.